<launch>
    
    <!-- Spawn Task-2 Models in Gazebo -->
    <include file="$(find pkg_vb_sim)/launch/task4_world.launch" />

    <include file = "$(find pkg_vb_sim)/launch/two_ur5_spawn.launch" >
        <arg name="arg_x" value="0.00" />
        <arg name="arg_y" value="0.00" />
        <arg name="arg_z" value="0.00" />
        <arg name="arg_R" value="0.0" />
        <arg name="arg_P" value="0.0" />
        <arg name="arg_Y" value="0.0" />
    </include>

    <!-- Spawn Boxes -->
    <node name= "task4_spawn_models" pkg= "pkg_vb_sim" type="task4_spawn_models.py"/>

    <!-- Run Conveyor Belt Service Server -->
    <node name= "node_service_server_conveyor_belt" pkg= "pkg_vb_sim" type="node_service_server_conveyor_belt.py" output="screen"/>
	
</launch>